### ISSUE TITLE
restrictions allow to remove point even if there is no free slot

### ISSUE THREAD

**Issue by haterzlin** [haterzlin/character-sheet#102]
Reproduction steps:
1. choose three disciplines
2. add two dots to first, one dot to second
3. remove one dot from first - I suppose this should not be possible
4. add 2 dots to third discipline - and this should not be possible too, but logic allows it
5. add fourth discipline with 2 dots

we should consider removing restriction enforcement

**Comment by longnicknam3** [comment]
no chce to asi zohladnit tie disciplinove sloty voci resource, nejak to zviazat
ja bych to mozno urobil aj tak ze to priradovanie by bolo natvrdo a na tie bodky by proste v character creation neslo klikat ale len pri lvl up
takze v momente ked v tom selectu vyberes disciplinu tak u prvej sa pevne nastavi 2 a u druhej sa pevne nastavi 1
mozno by potom v selectu mohol byt nejaky option na prehodenie (alebo ze by sa to nefiltrovalo, len by to bolo disablovane/oznacene inac, proste by to muselo byt uzivatelovi jasne ze ked na to klikne vedie to k nejakej pripadne destruktivnej akcii) alebo by tam mohol byt nejaky UI element prehodenia, alebo mozno by sa to dalo preusporiadat (to by mohlo byt zajimave aj keby to nemenilo hodnoty :))

**Comment by haterzlin** [comment]
no, ja neviem ako to vyriesit ked chcem zaroven zachovat, aby slo ubrat jeden bod...

takze si myslim, ze by to len ukazovalo, kolko este mozes a pripadne kolko XP to zozerie, tak by to bolo ok. Ale to je tazka zmena
